### DISCUSSION TITLE
Creating issues with a prefilled project

### DISCUSSION THREAD

**Discussion by acroca** [community/community#7460] [Projects and Issues]
Following [this guide](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-url-query), it says we can use `projects=` to set a project in the new issue page, but it doesn't seem to work for new projects (beta).

**Comment by azenMatt** [comment]
Hi @acroca 👋  I recognise that handle! I hope you have been well! 

This isn't something we support at the moment, but it is on our list of 'parity' items that we plan on bringing over from the old projects to the new projects before we GA it. We are working through these at the moment and will keep you updated when we have this one. 

Right now, we have work in flight for: 

- Project description 
- Outside collaborator support
- Integration with issue / pull request timelines 
- Index page filtering

**Reply by acroca** [reply]
Thanks for your answer @azenMatt! Helpful as usual 👍

**Reply by thromera** [reply]
Hi! Are there any updates on this?

I'm trying to have an URL that prefills the projects. The project is from the Kanban view (so the new project "beta"?). 

Maybe I'm missing something, please let me know if that's the case! 
- The project URL I have is https://github.com/orgs/xxxxx/projects/61
- The URL to prefill a new issue: https://github.com/xxx/yyy/issues/new?template=bug_report.md&projects=xxx/61&labels=Key+Accounts,bug&milestone=247

Thanks!

**Reply by tomdcc** [reply]
Just commenting in the hope that it gets some attention. Any movement here? We use links / bookmarks to create issues and ensure that they're put into the correct project to get attention and contributors don't always get this right.
